src/hotspot/share/runtime/globals.hpp

```diff
@@ -2009,6 +2009,11 @@ const int ObjectAlignmentInBytes = 8;
                                                                             \
   product(bool, ProfileExceptionHandlers, true,                             \
           "Profile exception handlers")                                     \
+                                                                            \
+  product(bool, AlwaysRecordEvolDependencies, true, DIAGNOSTIC,             \
+                "Unconditionally record method dependencies on class "      \
```

Comment:
"... record compiled method dependencies ..."?

Reply:
(yes, “compiled methods” or even “nmethods”, or “methods in code cache”)

Reply:
Thanks.  I went with "nmethods" because it's the shortest alternative :)